src/node_crypto.h

```diff
@@ -527,7 +531,11 @@ class Hash : public BaseObject {
   ~Hash() override {
     if (!initialised_)
       return;
-    EVP_MD_CTX_cleanup(&mdctx_);
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+    free(mdctx_);
```

Comment:
This leaks memory. It does not call `EVP_MD_CTX_cleanup`. But you're better off calling `EVP_MD_CTX_destroy` here and changing the below to call `EVP_MD_CTX_create`.